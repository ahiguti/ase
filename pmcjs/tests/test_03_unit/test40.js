
x = function(a, b) { return a + b; }
var y = Perl.CoerceToFunction(x);
print(x);
print(y);
var r = y(10, 30);
print("r=" + r);

