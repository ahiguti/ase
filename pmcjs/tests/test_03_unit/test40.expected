function (a, b) {
    return a + b;
}
function () {
    [native code]
}
r=40
